Lib/asyncio/queues.py

```diff
@@ -44,7 +44,7 @@ def __init__(self, maxsize=0):
 
```

Comment:
You did not change the call site.

Reply:
I missed this, thanks! Updated in commit 8b3df393ec241c11210bf54f5c5669c3ad1b5796